### PR TITLE
Add Chinese Translation (2022.7.31)

### PR DESCRIPTION
**Pull request by Bill-Haku**
Translation include:
- All new vocabularies updated this week
- *2 & *3 in `foods.json5`

Note: I found that some expressions in Traditional Chinese did not use the corresponding character in Simplified Chinese in some kinds of foods, although the word may appeared in other places and used the corresponding characters. zhTW expression have been recorded in notes, it may be helpful when supporting Traditional Chinese.

### CODE REVIEW

**Comment by Bill-Haku** [comment]
@xicri I'm sorry but I'm wondering where can I find the `invalid character '\n' at 312:0`? I've checked the line 312 in all files but it still failed.

File: dataset/dictionary/foods.json5 [hunk 1]
```diff
@@ -142,175 +142,205 @@
...
     en: "Radish and Fish Stew",
     ja: "魚とダイコンの煮込み",
+    zhCN: "鲜鱼炖萝卜",
     pronunciationJa: "さかなとダイコンのにこみ",
     tags: [ "food" ],
   },
   {
     en: "Berry Mizu Manjuu",
     ja: "ラスベリー水まんじゅう",
+    zhCN: "树莓水馒头,
```
Comment (xicri) [inline dataset/dictionary/foods.json5:311]: You need a doublequote here and I think this is the cause of the error.
```suggestion
    zhCN: "树莓水馒头",
```

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/foods.json5 (hunk 1 above)
Comment (Bill-Haku) [reply dataset/dictionary/foods.json5:311]: Oh thank you!

File: dataset/dictionary/organizations.json5 [hunk 2]
```diff
@@ -386,17 +387,22 @@
...
     pronunciationJa: "エルマイトりょだん",
     tags: [ "sumeru", "organization" ],
   },
   {
     en: "Amurta",
     ja: "生論派",
+    zhCN: "生论派",
     tags: [ "sumeru", "organization" ],
+    // Note: I don't think tag "organization" is suitable for this and the following.
+    // Because they are just academic genres.
```
Comment (xicri) [inline dataset/dictionary/organizations.json5:400]: You are right.
Can you remove `organization` tag from Amurta and Haravatat, and put them before Akasha in story.json5?

**Review by xicri — CHANGES_REQUESTED** [review]

File: dataset/dictionary/organizations.json5 (hunk 2 above)
Comment (Bill-Haku) [reply dataset/dictionary/organizations.json5:400]: OK

**Review by xicri — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**